UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/FulfilmentDataService.cs

```diff
@@ -26,6 +26,17 @@ public class FulfilmentDataService : IFulfilmentDataService
         private const string UPDATEZIPEXCHANGESETFILEEXTENSION = "zip";
         private const string CATALOGUEFILEEXTENSION = "xml";
         private const string ENCUPDATELISTFILEEXTENSION = "csv";
+
+        private readonly Dictionary<string, string> mimeTypes = new()
+        {
+            { ".zip", "application/zip" },
+            { ".xml", "text/xml" },
+            { ".csv", "text/csv" },
+            { ".iso", "application/octet-stream" },
+            { ".sha1", "application/octet-stream" }
+        };
+        private readonly string DEFAULTMIMETYPE = "application/octet-stream";
+
         private readonly string _homeDirectoryPath = string.Empty;
```

Comment:
_homeDirectoryPath doesn't need initialising to string.Empty because it's read-only and set in the constructor

Reply:
Updated